Makefile

```diff
@@ -319,9 +318,7 @@ ZPKG_DEPENDS += $(APP_BUILD_DIR)/extensions
 ZPKG_DEPENDS += $(DOCS_HTML_DIR)
 
 $(APP_BUILD_DIR)/ipmi.ini: $(APP_FILE)
-	$(PYTHON) -m common.python.copy_file_in_modules \
-        --fallback $(TOP)/common/templates/default_ipmi.ini \
-        $(TOP) $< ipmi.ini $@
+	$(PYTHON) -m common.python.make_ipmi_ini $(TOP) $< $@
```

Comment:
I renamed `make_ipmi_ini` in master to make it reusable and to pull out the content of default `ipmi.ini` to a file (making it easier to maintain), not sure why this is changed in this pull request

Reply:
oops, yes I see now how this got dropped. Yes, it caused an error - but hopefully fixed now!